deeplearning4j-keras/src/main/java/org/deeplearning4j/keras/RecursiveCopier.java

```diff
@@ -21,7 +27,7 @@ public void copy() {
 
     private void copyRecursive(int shapeIdx, int[] indexes) {
         if (shape.length == shapeIdx) {
-            output.putScalar(indexes, input[inputIdx++]);
+            output.putScalar(indexes, input.getFloat(inputIdx++));
```

Comment:
not sure if i'm not doing something colossally stupid, but maybe instead of using the *Copier I could just do: 
```
            INDArray input = Nd4j.create(shape);
            input.setData(dataBuffer);
```